extensions/database/module/scripts/index/database-import-form.html

```diff
@@ -29,44 +29,43 @@
-                                       </div>
-                                        <div class="pure-control-group">
-                                           <label id="databasePasswordLabel" for="databasePassword">Database Password:</label>
-                                           <input bind="databasePasswordInput" id="databasePassword" type="password" placeholder="Enter Database Password" class="pure-input-1-3" required>
-                                       </div>
-                                        <div class="pure-control-group">
-                                           <label id="databaseNameLabel" for="initialDatabase">Database Name:</label>
-                                           <input bind="initialDatabaseInput" id="initialDatabase" type="text" placeholder="Enter Database" class="pure-input-1-3" required>
-                                       </div>
-                                       <div class="pure-control-group" style="display:none;">
-                                           <label id="databaseSchemaLabel" for="initialSchema">Schema Name:</label>
-                                           <input bind="initialSchemaInput" id="initialSchema" type="text" placeholder="Enter Initial Schema" class="pure-input-1-3" required>
-                                       </div>
+
+                                        <div class="pure-control-group dbtype-options dbt-postgresql dbt-mysql dbt-mariadb">
+                                          <label id="databaseHostLabel" for="databaseHost">Database Host:</label> 
+                                          <input bind="databaseHostInput" id="databaseHost" type="text" placeholder="" value="localhost" class="pure-input-1-3" required>
+                                        </div>
+                                        <div class="pure-control-group dbtype-options dbt-postgresql dbt-mysql dbt-mariadb">
+                                          <label id="databasePortLabel" for="databasePort">Database Port:</label> 
+                                          <input bind="databasePortInput" id="databasePort" type="text" placeholder="" value="5432" class="pure-input-1-3" required>
+                                        </div>
+                                        <div class="pure-control-group dbtype-options dbt-postgresql dbt-mysql dbt-mariadb">
+                                          <label id="databaseUserLabel" for="databaseUser">Database User:</label> 
+                                          <input bind="databaseUserInput" id="databaseUser" type="text" placeholder="" value="postgres" class="pure-input-1-3" required>
+                                        </div>
+                                        <div class="pure-control-group dbtype-options dbt-postgresql dbt-mysql dbt-mariadb">
+                                          <label id="databasePasswordLabel" for="databasePassword">Database Password:</label> 
```

Comment:
How about we remove this text from the HTML file, since it is overwritten by i18n in Javascript anyway?

Reply:
Done.